aws_lambda_powertools/logging/logger.py

```diff
@@ -443,6 +445,18 @@ def decorate(event, context, *args, **kwargs):
 
         return decorate
 
+    def setLevel(self, level: Union[str, int]):
+        """
+        Set the logging level for the logger.
+
+        Parameters:
+        -----------
+        level str | int
+            The level to set. Can be a string representing the level name: 'DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'
+            or an integer representing the level value: 10 for 'DEBUG', 20 for 'INFO', 30 for 'WARNING', 40 for 'ERROR', 50 for 'CRITICAL'. # noqa: E501
+        """
+        self._logger.setLevel(level)
```

Comment:
we also need to update our custom attribute: `self.log_level` here.

Reply:
Yes, I missed it. Fixed.